.circleci/config.yml

```diff
@@ -21,7 +21,13 @@ jobs:
           key: v1-dependencies-{{ checksum "package.json" }}
       - run:
           name: Audit
-          command: npm audit --audit-level=low
+          command: npm audit --audit-level=low --production
```

Comment:
here's the audit change. in practice, this should be safe because we only use the dev dependencies for building+testing

Reply:
hmm, there are non-dev dependancies though:

```
"dependencies": {
    "@babel/parser": "^7.0.0",	
    "@babel/types": "^7.0.0"
  },
```

These should be dev deps.

FWIW, agreed that consumers of the plugin aren't at risk for issues in dev dependancies, but contributors could be. It makes sense to apply the `--production` flag for deployments but for future work we should consider a scheduled job that applies to all dependancies.

Reply:
Agreed that consumers of the plugin aren't at risk for issues in dev dependancies, but contributors could be. It makes sense to apply the `--production` flag for deployments but for future work we should consider a scheduled job that applies to all dependancies.

Reply:
That's true. Once we can update `babel-cli` we can switch it back to checking all deps.